constants/assets.js

```diff
@@ -1,3 +1,7 @@
-// eslint-disable-next-line import/prefer-default-export
 export const FAVICON_URL =
   'https://res.cloudinary.com/wir-by/image/upload/v1567534508/production/favicon/favicon.png';
+
+export const getAppleTouchIcon = isProd =>
```

Comment:
```js
export const APPLE_TOUCH_ICON = __PROD__ ? ...
```

Reply:
why don't we use the same approach for `getGoogleAnalyticsID()` then?